doc/sphinx/source/graph-basics.rst

```diff
@@ -1,25 +1,93 @@
 .. graph-basics:
 
+#############
 Graph Basics
-============
+#############
 
-This section describes the basic concepts you need to know when 
-constructing a graph.
 
+This section explains some of the primary concepts used in the nGraph Library.
+Some new ideas regarding our unique departure from the first generation of deep 
+learning software design are also introduced. 
 
-Framework Bridges
-------------------
+Many frameworks have a baked-in assumption that performant deep learning systems 
+require stacks of GPUs. Given this assumption, the frameworks tend to yield best 
+performance `only when given full control`_ of the GPU devices, and if 
+`any other resource demands`_ are made on the GPU, the GPU ends up performing about 
+the same as a CPU. Indeed, many GPU manufacturers have gone out of their way to 
+boast of multi-GPU to CPU performance, which is essentially an apples-to-oranges 
+comparison for this very reason. 
 
-Frontends (or users who require the flexibility of constructing 
-Ops directly) can utilize a set of graph construction functions 
-to construct graphs. 
+As different kinds of deep learning applications emerge, and as layers of 
```

Comment:
I think I would rewrite this and the preceding paragraph along roughly the following lines (the below needs considerable wordsmithing):

1. The historical dominance of GPUs at the beginning of the current deep learning boom means that many framework authors made GPU-specific design decisions at a very deep level. (For example, they want to own memory allocation, they assume that the compute device is a GPU over which they have complete control and which they do not need to share, they tend to write things in a SIMT-friendly manner (see https://en.wikipedia.org/wiki/Single_instruction,_multiple_threads) even if this is not appropriate for some hardware targets, and they may support only a limited set of data layout conventions.)
2. The shift toward greater diversity in deep learning hardware requires these assumptions to be revisited, but incorporating direct support for all of the different hardware targets out there, each of which has its own preferences when it comes to the above factors, is a very heavy burden on framework owners.
3. nGraph lightens that burden by raising the abstraction level, and letting hardware-specific backends make these decisions automatically, taking into account the needs of each target hardware platform to achieve maximum performance.
4. This makes things easier on framework owners, but also (as new models are developed) on data scientists, who will not have to keep in mind nearly as many low-level hardware details when architecting their models.